components/actionbutton/index.css

```diff
@@ -114,7 +74,6 @@
 		--mod-actionbutton-border-color-focus: var(--spectrum-transparent-black-600);
 
 		--spectrum-actionbutton-background-color-disabled: transparent;
-		--spectrum-actionbutton-background-color-selected-disabled: var(--spectrum-transparent-black-200);
```

Comment:
Do we need this line still? It looks like the static white has both a disabled and a selected-disabled custom property, so maybe we still need both properties here. What do you think?

Reply:
@cdransf I thought we had added this back in for the static black, but we must have reverted it. That way both static white & static black has their own, specific `--spectrum-actionbutton-background-color-selected-disabled` value.